svir/dialogs/load_gmf_data_as_layer_dialog.py

```diff
@@ -69,11 +69,15 @@ def __init__(self, drive_engine_dlg, iface, viewer_dock, session, hostname,
             self.on_extract_error)
         QgsApplication.taskManager().addTask(self.extract_npz_task)
 
+    def get_closest_element(self, element, elements):
+        return min(elements, key=lambda x: abs(x - element))
+
```

Comment:
The idea is good, but you need to use numpy for performance:
```
return elements[numpy.abs(elements - element).argmin()]        
```

Reply:
Thanks. I updated it as suggested.